### DISCUSSION TITLE
When does opensea and uniswap will migrate to v2 ?

### DISCUSSION THREAD

**Discussion by tapan-cn** [WalletConnect/walletconnect-monorepo#2014] [Developer Support] [answered]
I am creating a wallet app with wallet connect support. I integrated the v2 version into my wallet app but when I tried to test with popular platforms like uniswap or openseas but did not worked .

Kotlin sdk shows when i scanned open sea or uniswap it says invalid url seems like opensea and uniswap is not updated to v2.

Does anyone know any time line for these apps to update from v1 to v2?

Does any popular is upgraded with v2 or everybody is yet to update to v2?

Which version should I use if I am starting a new app at the moment in case of nobody has upgraded?

**Comment by 95decode** [comment]
Originally, walletConnect v1.0 Shutdown scheduled for March 1,

But the shutdown was extended to June 28 (read about it [here](https://medium.com/walletconnect/weve-reset-the-clock-on-the-walletconnect-v1-0-shutdown-now-scheduled-for-june-28-2023-ead2d953b595))

It seems that you need to make both v1 and v2 compatible by June 28, before all dapps migrate to v2.

**Comment by tapan-cn** [comment]
Thanks for reply @95decode 
But the main problem is testing, As I could not find any dApp migrated to v2 (a part from the sample apps) to test real-world scenarios for my app

**Reply by finessevanes** [reply, marked as answer]
https://www.stellarx.com/
https://friktion.fi/

For a list: https://explorer.walletconnect.com/?type=dapp&version=2

**Comment by finessevanes** [comment]
https://www.stellarx.com/
https://friktion.fi/

For a list: https://explorer.walletconnect.com/?type=dapp&version=2

**Reply by tapan-cn** [reply]
thanks

**Comment by Gary744** [comment]
Thank you!

**Comment by mishchenkoy** [comment]
👍👍👍

**Comment by Scottie33Pippen** [comment]
when does anyone know?
